packages/core/src/components/ListContainer/ListItem/ListItem.tsx

```diff
@@ -0,0 +1,224 @@
+   */
+  startAdornment?: React.ReactNode;
+  /**
+   * Element placed after the children and aligned next to the margin.
+   * Also removes the right padding (gutter).
+   *
+   * Some modifications are applied, assuming that it is an icon (changing the color when the item is disabled).
+   * If unwanted, the element should be placed directly as a child.
+   */
+  endAdornment?: React.ReactNode;
+  /** A Jss Object used to override or extend the styles applied to the empty state component. */
+  classes?: {
+    root?: string;
+    focus?: string;
+    startAdornment?: string;
+    endAdornment?: string;
+    gutters?: string;
+    condensed?: string;
+    interactive?: string;
+    selected?: string;
+    disabled?: string;
+    withStartAdornment?: string;
+    withEndAdornment?: string;
+  };
+};
+
+const applyClassNameAndStateToElement = (
+  element,
+  selected,
+  disabled,
+  onClick,
+  className,
+  externalClassname
+) => {
+  if (element != null) {
+    return React.cloneElement(element, {
+      className: clsx(className, externalClassname, element?.props?.className),
+      checked: !!selected,
+      disabled,
+      onChange: (evt) => onClick?.(evt),
+    });
+  }
+
+  return null;
+};
+
+const applyClassNameToElement = (element, className, externalClassname) => {
+  if (element != null) {
+    return React.cloneElement(element, {
+      className: clsx(className, externalClassname, element?.props?.className),
+    });
+  }
+
+  return null;
+};
+
+/**
+ * ListItem description/documentation paragraph
+ */
+export const HvListItem = ({
+  id,
+  classes,
+  className,
+  role,
+  selected,
+  disabled,
+  interactive: interactiveProp,
+  condensed: condensedProp,
+  disableGutters: disableGuttersProp,
+  startAdornment,
+  endAdornment,
+  onClick,
+  children,
+  tabIndex,
+  ...others
+}: HvListItemProps) => {
+  const {
+    topContainerRef,
+    condensed: condensedContext,
+    disableGutters: disableGuttersContext,
+    interactive: interactiveContext,
+  } = useContext<any>(HvListContext);
+
+  const condensed = condensedProp != null ? condensedProp : condensedContext;
+  const disableGutters =
+    disableGuttersProp != null ? disableGuttersProp : disableGuttersContext;
+  const interactive =
+    interactiveProp != null ? interactiveProp : interactiveContext;
+
+  const handleOnClick = useCallback(
+    (evt) => {
+      if (!disabled) {
+        onClick?.(evt);
+      }
+    },
+    [disabled, onClick]
+  );
+
+  const clonedStartAdornment = useMemo(
+    () =>
+      applyClassNameAndStateToElement(
+        startAdornment,
+        selected,
+        disabled,
+        handleOnClick,
+        clsx("startAdornment", disabled && "disabled"),
+        classes?.startAdornment
+      ),
+    [classes?.startAdornment, disabled, handleOnClick, selected, startAdornment]
+  );
+  const clonedEndAdornment = useMemo(
+    () =>
+      applyClassNameToElement(
+        endAdornment,
+        "endAdornment",
```

Comment:
We should also change this to `clsx("endAdornment", disabled && "disabled")` in order for the end adornment to change its color when disabled.